src/main/java/menion/android/whereyougo/preferences/PreferenceValues.java

```diff
@@ -181,11 +181,7 @@ public static String getLanguageCode() {
             lang = java.util.Locale.getDefault().getLanguage();
         }
         Logger.w(TAG, "getLanguageCode() - " + lang);
```

Comment:
remove the lines:
~~~
        String lang = Preferences.getStringPreference(R.string.pref_KEY_S_LANGUAGE);
        if ("".equals(lang) || "default".equals(lang)) {
            lang = java.util.Locale.getDefault().getLanguage();
        }
        Logger.w(TAG, "getLanguageCode() - " + lang);
~~~
add lines
~~~
        // language for info dialog html files
        // only english supported
        Logger.w(TAG, "getLanguageCode() - " + "en");
~~~